### PR TITLE
Various edits for punctuation, grammar, spelling, etc.

### PR DESCRIPTION
**Pull request by L2G**
I've been accumulating small edits to improve the English in Homebrew's prompts, documentation, and code comments. This is a single commit that contains all such changes that I've collected to date.

If it would be better for me to divide these into multiple commits along some kind of conceptual lines (commands vs. libraries, prompts vs. code comments, etc.), let me know.

### CODE REVIEW

**Comment by DomT4** [comment]
You need to `brew man` here as well, since you edited the Markdown manpage but not the generated one.

File: share/doc/homebrew/Acceptable-Formulae.md [hunk 1]
```diff
@@ -101,7 +101,7 @@ Please trust that we need to use our discretion based on our experience
 running a package manager.
 
 ### Stuff that builds a .app
-Don’t make your formula build an `.app` (native OS X Application), we
+Don’t make your formula build an `.app` (native OS X Application); we
```
Comment (bfontaine) [inline share/doc/homebrew/Acceptable-Formulae.md]: Why this change? The sentence feels more natural to me with the comma.

Comment (tdsmith) [inline share/doc/homebrew/Acceptable-Formulae.md]: The comma creates a comma splice, joining two complete sentences, which is incorrect.

Comment (L2G) [inline share/doc/homebrew/Acceptable-Formulae.md]: http://writing.wisc.edu/Handbook/Semicolons.html

**Comment by tdsmith** [comment]
The changes look good to me; thanks!

File: share/doc/homebrew/Acceptable-Formulae.md (hunk 1 above)
Comment (bfontaine) [inline share/doc/homebrew/Acceptable-Formulae.md]: Thanks Tim, I didn’t know about this rule; this is something that doesn’t exist in French.

File: Library/Contributions/brew_zsh_completion.zsh [hunk 2]
```diff
@@ -31,7 +31,7 @@ _1st_arguments=(
   'commands:show a list of commands'
   'config:show homebrew and system configuration'
   'create:create a new formula'
-  'deps:list dependencies and dependants of a formula'
+  'deps:list dependencies and dependents of a formula'
```
Comment (mikemcquaid) [inline Library/Contributions/brew_zsh_completion.zsh]: Both spellings are valid so let's leave this as-is.

File: Library/Homebrew/manpages/brew.1.md [hunk 3]
```diff
@@ -236,7 +236,7 @@ Note that these flags should only appear after a command.
 
   * `ln`, `link [--overwrite] [--dry-run] [--force]` <formula>:
     Symlink all of <formula>'s installed files into the Homebrew prefix. This
-    is done automatically when you install formula, but can be useful for DIY
+    is done automatically when you install formulae, but can be useful for DIY
```
Comment (mikemcquaid) [inline Library/Homebrew/manpages/brew.1.md]: can probably kill the comma here (I know you didn't add it)

**Comment by mikemcquaid** [comment]
A few comments but otherwise looks good.

**Comment by DomT4** [comment]
> "dependant" is a totally kosher spelling for this. So sayeth the OSX dictionary, anyway.

It's an American/British [thing](http://www.grammar-monster.com/easily_confused/dependant_dependent.htm).

File: Library/Homebrew/cmd/audit.rb [hunk 4]
```diff
@@ -195,12 +195,12 @@ def audit_formula_name
...
 
     if @@aliases.include? name
-      problem "Formula name is conflicted with existed aliases."
+      problem "Formula name conflicts with existing aliases."
       return
     end
 
     if !formula.core_formula? && Formula.core_names.include?(name)
-      problem "Formula name is conflicted with existed core formula."
+      problem "Formula name conflicts with existing core formulae."
```
Comment (DomT4) [inline Library/Homebrew/cmd/audit.rb]: This is a really small nit, but this should probably remain `core formula` rather than `core formulae.` - The latter is the plural, but the audit will only address one formula at a time.

**Comment by DomT4** [comment]
Seemed to be no further objections/issues on this, so merged in https://github.com/Homebrew/homebrew/commit/ace0d8f92a361266a44c970a6601fa1437e9327a. Thanks for your contribution to Homebrew, we appreciate it! :smiley_cat:
